lab1.js

```diff
@@ -133,7 +153,24 @@ var tooHungryDay;
  meals)
 */
 
-// your code goes here
+var j = 1;
+var totalMeals = mealsPerDay[0];
+var mealAvg = 0;
+var avgThreshold = true;
+
+do {
```

Comment:
something's off in here - you're ending up with a tooHungryLion on day 5, instead of day 7. Yes, the asserts pass,  but it also doesn't pass the "mental math on the elements of the array" test. :)